Code/Framework/AzToolsFramework/AzToolsFramework/ActionManager/Menu/EditorMenu.h

```diff
@@ -37,7 +37,7 @@ namespace AzToolsFramework
         AZ_RTTI(EditorMenu, "{6B6F6802-C587-4734-A5DB-5732329EED03}");
 
         EditorMenu();
-        explicit EditorMenu(const AZStd::string& name);
+        explicit EditorMenu(const AZStd::string& identifier, const AZStd::string& name);
```

Comment:
suggestion: We could remove the `explicit` constructor here now we have two parameters (and no default arguments).